pkg/k8sutil/metadata.go

```diff
@@ -35,6 +35,22 @@ func NodeAnnotationCondition(selector fields.Selector) watch.ConditionFunc {
 	}
 }
 
+// GetNodeRetry gets a node object, retrying up to DefaultBackoff number of times if it fails
+func GetNodeRetry(nc v1core.NodeInterface, node string) (*v1api.Node, error) {
+	var apiNode *v1api.Node
+	err := RetryOnError(DefaultBackoff, func() error {
+		n, getErr := nc.Get(node, v1meta.GetOptions{})
```

Comment:
The function literal is a new scope, so it's ok to shadow `err` here.

Reply:
true, just matching convention: https://github.com/dollarshaveclub/container-linux-update-operator/blob/c11ab1407f42268463a74096320ec17da96f823b/pkg/k8sutil/metadata.go#L61